modules/util/portal-tools-service-builder/src/com/liferay/portal/tools/service/builder/dependencies/persistence_impl_finder_find.ftl

```diff
@@ -1179,6 +1157,84 @@ that may or may not be enforced with a unique index at the database level. Case
 					</#if>
 				</#list>
 
+				<#if finder.hasColumn("groupId") && finder.getColumn("groupId").hasArrayableOperator()>
+
+					<#list finderColsList as finderCol>
+						<#if finderCol.name == "groupId">
+
+							List<Long> enabledGroupIdsList = new ArrayList<Long>();
+							List<Long> notEnabledGroupIdsList = new ArrayList<Long>();
+
+							for (${finderCol.type} ${finderCol.name} : ${finderCol.names}) {
+								if (!InlineSQLHelperUtil.isEnabled(${finderCol.name})) {
+									notEnabledGroupIdsList.add(${finderCol.name});
+								}
+								else {
+									enabledGroupIdsList.add(${finderCol.name});
+								}
+							}
+
+							List<${entity.name}> list = new ArrayList<${entity.name}>();
```

Comment:
Hi Sam,

Do you mean "List<>" is not good or "list" is not good? Maybe should I use
xxxList?

Bejond Shao

Jr. Engineer, Technical Support

Subscription Services

Liferay Dalian Software Co., Ltd.

On Wed, Sep 2, 2015 at 5:49 PM, Samuel Kong <notifications@github.com>
wrote:

> In
> modules/util/portal-tools-service-builder/src/com/liferay/portal/tools/service/builder/dependencies/persistence_impl_finder_find.ftl
> <https://github.com/samuelkong/liferay-portal/pull/710#discussion_r38514942>
> :
>
> > +					<#list finderColsList as finderCol>
> > +						<#if finderCol.name == "groupId">
> > +
> > +							List<Long> enabledGroupIdsList = new ArrayList<Long>();
> > +							List<Long> notEnabledGroupIdsList = new ArrayList<Long>();
> > +
> > +							for (${finderCol.type} ${finderCol.name} : ${finderCol.names}) {
> > +								if (!InlineSQLHelperUtil.isEnabled(${finderCol.name})) {
> > +									notEnabledGroupIdsList.add(${finderCol.name});
> > +								}
> > +								else {
> > +									enabledGroupIdsList.add(${finderCol.name});
> > +								}
> > +							}
> > +
> > +							List<${entity.name}> list = new ArrayList<${entity.name}>();
>
> As far as variable naming, list is not going to get approved by Brian.
> Maybe something like ${entity.name}s
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/samuelkong/liferay-portal/pull/710/files#r38514942>.
>